code/modules/surgery/organs/augments_arms.dm

```diff
@@ -559,3 +559,35 @@
 		qdel(src)
 		qdel(I)
 	return ..()
+
+/obj/item/organ/internal/cyberimp/arm/muscle
+	name = "Strong-arm empowered musculature implant"
+	desc = "When implanted, this cybernetic implant will enhance the muscles of the arm to deliver more power-per-action. Only has to be installed in one arm."
+	icon_state = "muscle_imp"
+
+	parent_organ = "l_arm" //Left arm by default
+	slot = "l_arm_device"
+
+	actions_types = list()
+
+/obj/item/organ/internal/cyberimp/arm/muscle/insert(mob/living/carbon/M, special, dont_remove_slot)
+	. = ..()
+	var/datum/martial_art/muscle_implant/muscle_implant = new(null)
+	muscle_implant.teach(M)
+
+/obj/item/organ/internal/cyberimp/arm/muscle/remove(mob/living/carbon/M, special)
+	. = ..()
+	var/datum/martial_art/muscle_implant/muscle_implant = new(null)
```

Comment:
muscle_implant should be a var on /obj/item/organ/internal/cyberimp/arm/muscle, and shouldnt make a second one on remove.